File: Segundo-Bimestre/Parcial 1/OperacionesMatrices/OperacionesMatrices_java/src/operaciones/matrices/SumaMatrices.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package operaciones.matrices;
import java.util.Scanner;
/**
 *
 * @author DELL
 */
public class SumaMatrices {
    public static void main(String[] args) {
        Scanner leer = new Scanner (System.in);

        // Declarar e inicializar las variables
        int i, j, n;
        n = 0; 
        
        // Ingresar el tamaño de las matrices
        System.out.println("***SUMA DE MATRICES***");
        System.out.print("Ingrese el tamaño de las matrices a ingresar: ");
        n = leer.nextInt();
        
        // Declarar la matriz e ingresar sus valores
        int [][] A = new int [n][n];
        int [][] B = new int [n][n];
        
        System.out.println("\nIngresar los elementos de la matriz A");
        for (i=0; i<n; i++) {
            for (j=0; j<n; j++) {
                System.out.print("A ["+i+"] ["+j+"] = ");
                A[i][j] = leer.nextInt();
            }
        }

        System.out.println("\nIngresar los elementos de la matriz B");
        for (i=0; i<n; i++) {
            for (j=0; j<n; j++) {
                System.out.print("B ["+i+"] ["+j+"] = ");
                B[i][j] = leer.nextInt();
            }
        }
        
        System.out.println("\nLa matriz A es: ");
        for (i=0; i<n; i++) {
            for (j=0; j<n; j++) {
                System.out.print(A[i][j]+"   ");
            }
            System.out.println("");
        } 

        System.out.println("\nLa matriz B es: ");
        for (i=0; i<n; i++) {
            for (j=0; j<n; j++) {
                System.out.print(B[i][j]+"   ");
            }
            System.out.println("");
        }
        
        // Suma de matrices

        int [][] C = new int [n][n];
        
        for (i=0;i<n;i++) {
            for (j=0;j<n;j++) {
                C[i][j] = A[i][j] + B [i][j];
            }
        }
        
        System.out.println("\n***La matriz C (suma de la matriz A y B) es: ***");
        for (i=0; i<n; i++) {
            for (j=0; j<n; j++) {
                System.out.print(C[i][j]+"   ");
            }
            System.out.println("");
        } 
    }
}

File: Segundo-Bimestre/Parcial 1/OperacionesMatrices/OperacionesMatrices_python/SumaMatrices.py
print("")
print("***SUMA DE MATRICES***")
n = int(input("Ingrese el tamaño de las matrices a ingresar: "))

# Declarar la matriz
A = []
B = []

print("\nIngresar los elementos de la matriz A")
for i in range (0, n):
    A.append([])
    for j in range (0, n):
        valor = int(input("A["+str(i)+"]["+str(j)+"]: "))
        A[i].append(valor)

print("\nIngresar los elementos de la matriz B")
for i in range (0, n):
    B.append([])
    for j in range (0, n):
        valor = int(input("B["+str(i)+"]["+str(j)+"]: "))
        B[i].append(valor)

print("\nLa matriz A es: ")
for i in range (0, n):
    for j in range (0, n):
        print(A[i][j], end="   ")
    print("")

print("\nLa matriz B es: ")
for i in range (0, n):
    for j in range (0, n):
        print(B[i][j], end="   ")
    print("")

# Suma de matrices
C = []

for i in range (0, n):
    C.append([])
    for j in range (0, n):
        valor = A[i][j] + B[i][j]
        C[i].append(valor)

print("\nLa matriz C (Suma de la matriz A y B) es: ")
for i in range (0, n):
    for j in range (0, n):
        print(C[i][j], end="   ")
    print("")